docs/keycodes.md

```diff
@@ -0,0 +1,299 @@
+|-----------------------|---------------------------------------------------------------------|
+|`KC.RESET`             |Put the keyboard into DFU mode for flashing                          |
+|`KC.DEBUG`             |Toggle debug mode                                                    |
+|`KC.GESC`              |Escape when tapped, <code>&#96;</code> when pressed with Shift or GUI|
+|`KC.LEAD`              |The [Leader key]                                                     |
+|`KC.UC_MODE_NOOP`      |Sets UnicodeMode to NOOP                                             |
+|`KC.UC_MODE_LINUX`     |Sets UnicodeMode to NOOP                                             |
+|`KC.UC_MODE_MACOS`     |Sets UnicodeMode to NOOP                                             |
+|`KC.UC_MODE_WINC`      |Sets UnicodeMode to NOOP                                             |
+|`KC.MACRO_SLEEP_MS(ms)`|Sleeps in a macro. Check MACROS for more information.                |
+
+
+## [Layer Switching]
+
+|Key         |Description                                                                  |
+|-----------------|------------------------------------------------------------------------|
+|`KC.DF(layer)`      |Switches the default layer                                           |
+|`KC.MO(layer)`      |Momentarily activates layer, switches off when you let go            |
+|`KC.LM(layer, mod)` |As `MO(layer)` but with `mod` active                                 |
+|`KC.LT(layer, kc)`  |Momentarily activates layer if held, sends kc if tapped              |
+|`KC.TG(layer)`      |Toggles the layer (enables it if no active, and vise versa)          |
+|`KC.TO(layer)`      |Activates layer and deactivates all other layers                     |
+|`KC.TT(layer)`      |Momentarily activates layer if held, toggles it if tapped repeatedly |
+
+
+## [Modifiers]
+
+|Key          |Description                                         |
+|-------------|----------------------------------------------------|
+|`KC.HYPR`    |Hold Left Control, Shift, Alt and GUI               |
+|`KC.MEH`     |Hold Left Control, Shift and Alt                    |
+|`KC.LCTL(kc)`|Hold Left Control and press `kc`                    |
+|`KC.LSFT(kc)`|Hold Left Shift and press `kc`                      |
+|`KC.LALT(kc)`|Hold Left Alt and press `kc`                        |
+|`KC.LGUI(kc)`|Hold Left GUI and press `kc`                        |
+|`KC.RCTL(kc)`|Hold Right Control and press `kc`                   |
+|`KC.RSFT(kc)`|Hold Right Shift and press `kc`                     |
+|`KC.RALT(kc)`|Hold Right Alt and press `kc`                       |
+|`KC.RGUI(kc)`|Hold Right GUI and press `kc`                       |
+|`KC.HYPR(kc)`|Hold Left Control, Shift, Alt and GUI and press `kc`|
+|`KC.MEH(kc)` |Hold Left Control, Shift and Alt and press `kc`     |
+|`KC.LCAG(kc)`|Hold Left Control, Alt and GUI and press `kc`       |
```

Comment:
We do not support this or anything below it within this section

Reply:
Can we?

Reply:
open a ticket for them, you can base the implementations off MEH and HYPER in #65 if you get to it before I do

Reply:
This is on the later list after tap dance as those fit as tap dance codes with a shortcode.